### ISSUE TITLE
Add custom pricing per node type

### ISSUE THREAD

**Issue by exolab** [opencost/opencost#1974]
We are using somewhere between 6-10 different node types on Hetzner for hosting. AFAICT it is currently not possible to use OpenCost for meaningful cost monitoring in this scenario. Prices can be anywhere between 0.02 and 0.8 € / hour for CPU.

Being able to specify prices for each node type (could come from labels on the node) would help.

I saw a similar request [in this issue](https://github.com/opencost/opencost/issues/1398) but I believe the [proposed solution](https://github.com/opencost/opencost/pull/1480) does not really cover the use case.

**Comment by mattray** [comment]
Closing as a duplicate of https://github.com/opencost/opencost-website/issues/90
@daniwk said they had this working and we're still waiting on a solid example.

**Comment by exolab** [comment]
I have read all of those issues. It is still unclear to me whether what I want to do is possible with OpenCost at all. The default.json examples have two different prices, and it is totally unclear to me if I can add an arbitrary number of prices and how I would make sure they get used for the right nodes.

@daniwk: Would you mind sharing the essence of what you did in order to achieve this?
